modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -88,6 +96,10 @@ $product-menu-pages-administration-link-height: 72px;
 		margin-top: 20px;
 		padding-top: 16px;
 
+		a {
```

Comment:
don't you have to use .configure-link here? The class is added but not used anywhere

Reply:
👍